Patterns/English/English-DateTime.yaml

```diff
@@ -18,7 +18,7 @@ MonthNumRegex: !simpleRegex
 PeriodYearRegex: !simpleRegex
   def: \b(?<year>19\d{2}|20\d{2})\b
 WeekDayRegex: !simpleRegex
-  def: (?<weekday>Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Mon|Tue|Wedn|Weds|Wed|Thurs|Thur|Thu|Fri|Sat|Sun)\b
+  def: \b(?<weekday>Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Mon|Tue|Wedn|Weds|Wed|Thurs|Thur|Thu|Fri)\b

```

Comment:
Why remove Sat and Sun?

Reply:
"Sat" and "Sun" are ambiguous words. It is not correct to extract that alone. I use a new regex for the SingleWeekdayRegex which deleted the Sat and Sun.